tests/lib/rules/arrow-spaces.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * @fileoverview Tests for arrow-spaces
+ * @author Jxck
+ * @copyright 2015 Jxck. All rights reserved.
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+//
+
+var eslint = require("../../../lib/eslint"),
+    ESLintTester = require("eslint-tester");
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+var eslintTester = new ESLintTester(eslint);
+
+var TT = { before: true, after: true };
```

Comment:
The format of this test file deviates quite dramatically from our others, which will make it harder for others to use and modify. Can you switch to a simpler format?